actions/transfer-case.vbs

```diff
@@ -199,7 +199,7 @@ Do
     Do
         'Inside the do...loop for the calculate_button
         Dialog1 = ""
-        BeginDialog Dialog1, 0, 0, 346, 260, "Out-of-County Case Transfer for #" & MAXIS_case_number
+        BeginDialog Dialog1, 0, 0, 346, 260, "Out-of-County Case Transfer for # & MAXIS_case_number"
```

Comment:
Line 202 - The string will need to be adjusted to exclude the MAXIS_case_number. See the diff for the example.

Reply:
Good catch - reverted back to original.